common/config/rush/pnpm-lock.yaml

```diff
@@ -11763,7 +11763,7 @@ packages:
     dev: false
 
   file:projects/agrifood-farming.tgz:
-    resolution: {integrity: sha512-s8R4IIpQodHE0SQODEsc19cpaCtryR+cpzTKBPPxjgvHNyZl8IG1yykOLe0D/+3fICaApfqQtKVGnc5xqSQEpA==, tarball: file:projects/agrifood-farming.tgz}
+    resolution: {integrity: sha512-DKYhVpNWhs7+bnq5pISSj9p6UOFoxWBQMnRXcJLED0hl6jAeACL4Ltsm1IEQz3/be1Pr8+/9LUFH0N0l+ikxVA==, tarball: file:projects/agrifood-farming.tgz}
```

Comment:
looks a little strange, why we have this change? Could you merge latest main and then run rush update see if this change still exists?

Reply:
Rebased to main.